openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django models supporting the Comprehensive Theming subsystem
+"""
+from django.db import models
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
+    """
+    This is where the information about the site's theme gets stored to the db.
+
+    `site` field is foreignkey to django Site model
+    `theme_dir_name` contains directory name having Site's theme
+    """
+    site = models.ForeignKey(Site, related_name='themes')
```

Comment:
Tell me more about this stacking feature. Does that mean this model should have an `order` or `depth` field?